### ISSUE TITLE
final loss of pretrain stage 

### ISSUE THREAD

**Issue by weidong2018** [mlpc-ucsd/BLIVA#11]
Hi, nice work! Could you please provide the final loss (the loss log during the training process) of bliva in the pre-training stage?
